### PR TITLE
Cp 14449 add tests

### PR DESCRIPTION
**Pull request by roberthocking**
# Description of the issue
Adding smoke tests

# Description of changes
added small smoke test to check for exsistence of log groups

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
ran test on cloud9 instance

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make linter`

### CODE REVIEW

**Review by dmepham — COMMENTED** [review]
the structure looks good to me! the only comment I might have is that I wonder if test result logs might get lost if we run the `make destroy-eks-cluster` right after. IIRC the eksctl command is pretty verbose. maybe it would be clearer if test setup/teardown was separate from test execution? but I think either way works

File: Makefile [hunk 1]
```diff
@@ -309,3 +308,9 @@ init:
 		echo "Please download and install $(REQUIRED_GO_VERSION) using brew or other method of your choice"; \
 		exit 1; \
 	fi
+
+smoke-test:
+	go test -short -coverprofile smoke_coverage.txt -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
```
Comment (dmepham) [inline Makefile:313]: `smoke_coverage.txt` is generated on every run, right? is it worth adding to the .gitignore?

**Review by dmepham — APPROVED** [review]
lgtm! just one small comment. it might be nice to have a section in the README for our testing, but I'm not sure how that would work in a forked project with a bunch of non-CZ stuff in the README already.

File: Makefile (hunk 1 above)
Comment (roberthocking) [reply Makefile:313]: Good idea. I'll add it

**Pull request merged**